client/src/components/Overview.tsx

```diff
@@ -0,0 +1,359 @@
+          >
+            <Text as="h3" css={{ mb: '$2', fontSize: '$1', color: '$gray9' }}>
+              Total stacked
+            </Text>
+            <Heading size={'2xl'} css={{ color: '$orange11' }}>
+              {numberWithCommas(
+                microToStacks(statsData.stxStats.totalStacked).toFixed()
+              )}{' '}
+              <Box css={{ color: '$orange9' }} as="span">
+                STX
+              </Box>
+            </Heading>
+          </Box>
+          <Box
+            css={{
+              display: 'flex',
+              width: '50%',
+            }}
+          >
+            <Box
+              css={{
+                display: 'none',
+                width: 2,
+                height: 80,
+                backgroundColor: '$gray6',
+                '@md': {
+                  display: 'block',
+                },
+              }}
+            />
+            <Box css={{ '@md': { pl: '$3' } }}>
+              <Text as="h3" css={{ mb: '$2', fontSize: '$1', color: '$gray9' }}>
+                Next cycle (days)
+              </Text>
+              <Heading size={'2xl'} css={{ color: '$orange11' }}>
+                {getNextCycle(statsData.stxStats.nextCycleIn)}
+              </Heading>
+            </Box>
+          </Box>
+        </Flex>
+
+        <Flex
+          css={{
+            backgroundColor: '$gray2',
+            p: '$3',
+            borderRadius: '$2',
+            '@md': {
+              flex: 'auto',
+            },
+          }}
+          gap={'3'}
+          justify={'between'}
+          direction={{
+            '@initial': 'column',
+            '@lg': 'row',
+          }}
+        >
+          <Box>
+            <Text as="h3" css={{ mb: '$2', fontSize: '$1', color: '$gray9' }}>
+              BTC price
+            </Text>
+            <Flex gap={'2'}>
+              <Heading size={'2xl'} css={{ color: '$orange11' }}>
+                <Box css={{ color: '$orange9' }} as="span">
+                  $
+                </Box>
+                {numberWithCommas(statsData.priceData?.bitcoin.usd)}
+              </Heading>
+              <Text
+                css={{
+                  fontSize: '$2',
+                  color:
+                    statsData.priceData.bitcoin.usd_24h_change > 0
+                      ? '#23A77F'
+                      : '#E5484D',
+                }}
+              >
+                <Box
+                  as="span"
+                  css={{ display: 'inline-block', verticalAlign: 'middle' }}
+                >
+                  {statsData.priceData.bitcoin.usd_24h_change > 0 ? (
+                    <ArrowUpIcon style={{ color: '#23A77F' }} />
+                  ) : (
+                    <ArrowDownIcon style={{ color: '#E5484D' }} />
+                  )}
+                </Box>
+                {statsData.priceData.bitcoin.usd_24h_change.toFixed()}%
+              </Text>
+            </Flex>
+          </Box>
+          <Box
+            css={{
```

Comment:
added this property as a last resort after `flex: auto` and `flex: 1` were failing to stretch as would have been expected. Happy to try an alternative solution if you know another way the even sizing can be achieved :)